graph/src/components/versions/registry.rs

```diff
@@ -28,14 +28,14 @@ pub struct ApiVersion {
 
 impl ApiVersion {
     pub fn new(version_requirement: &VersionReq) -> Result<Self, String> {
-        let version = Self::resolve(&version_requirement)?;
+        let version = Self::resolve(version_requirement)?;
 
         Ok(Self {
             version: version.clone(),
             features: VERSION_COLLECTION
-                .get(&version)
-                .expect(format!("Version {:?} is not supported", version).as_str())
-                .to_vec(),
+                .get(version)
+                .unwrap_or_else(|| panic!("Version {:?} is not supported", version))
```

Comment:
Is this a manual change or clippy? Not sure why this is better?

Reply:
Automatic. It comes from [this commit](https://github.com/graphprotocol/graph-node/commit/271871081406a3f0ac245e4d9299c4433d7a5752) from the linter `clippy::expect_fun_call`.
As each rule has its own commit, we can drop the ones that are questionable.
[According to clippy](https://rust-lang.github.io/rust-clippy/stable/index.html#expect_fun_call), it was bad because `The function will always be called.`